modules/ossm-extensions-smextension-format.adoc

```diff
@@ -0,0 +1,62 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
+////
+:_content-type: REFERENCE
+[id="ossm-extensions-smextension-format_{context}"]
+= `ServiceMeshExtension` container format
+
+You must have a `.wasm` file containing the bytecode of your WebAssembly module, and a `manifest.yaml` file in the root of the container filesystem to make your container image a valid extension image.
```

Comment:
Though I didn't find this in ISG, I think there is more usage of "file system" in the docs than "filesystem", so this is just a suggestion for consistency. I leave the decision up to you 🙂 
```suggestion
You must have a `.wasm` file containing the bytecode of your WebAssembly module, and a `manifest.yaml` file in the root of the container file system to make your container image a valid extension image.
```